src/app/frontend/replicasetlist/replicasetcard.scss

```diff
@@ -1,19 +1,37 @@
-// Copyright 2015 Google Inc. All Rights Reserved.    
-//   
-// Licensed under the Apache License, Version 2.0 (the "License");   
-// you may not use this file except in compliance with the License.    
-// You may obtain a copy of the License at   
-//   
-//     http://www.apache.org/licenses/LICENSE-2.0    
-//   
-// Unless required by applicable law or agreed to in writing, software   
-// distributed under the License is distributed on an "AS IS" BASIS,   
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.    
-// See the License for the specific language governing permissions and   
-// limitations under the License.    
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
 
 @import '../../assets/styles/color_variables';
 
+$replicaset-deploy-btn: #f00;
+
+.md-fab {
+  &.kd-replicaset-deploy {
+    background-color: $replicaset-deploy-btn;
+    position: absolute;
+    right: 5px;
+    top: 28px;
+
+    // Needed to override default on hover color change for button
+    &:not([disabled]) {
+      &:hover {
```

Comment:
I really want to avoid this but as `!important` is not allowed then i don't see other option to override current rule.